serializer.go

```diff
@@ -22,10 +22,10 @@ func buildContentJson(fieldNames []string, fieldValues []string, sortedIndexes [
 			fieldDef := fields[fieldNames[index]]
 			escapedValue := escapeForJson(fieldValues[index])
 			if fieldDef.Cardinality == "n" {
-				if fieldDef.Datatype == "string" {
-					jsonPart = fmt.Sprintf(`"%s":%s`, fieldNames[index], toJsonArrayOfStr(escapedValue))
-				} else {
+				if fieldDef.Datatype == "integer" {
```

Comment:
I'm not sure that this is correct - I believe ORJ expects all types to be encoded as a string in the JSON, including integers. Does the result of this load successfully into ORJ?